ord_schema/validations.py

```diff
@@ -3,10 +3,17 @@
 import math
 import re
 import warnings
+from urllib import request
+from urllib.error import HTTPError
 from dateutil import parser
 
 from ord_schema.proto import reaction_pb2
 
+try:
+    from rdkit import Chem, __version__ as RDKIT_VERSION
```

Comment:
Please put these imports on separate lines, even though they're from the same package; see [here](http://google.github.io/styleguide/pyguide.html#313-imports-formatting).

Reply:
Done